spec/controllers/idv/capture_doc_status_controller_spec.rb

```diff
@@ -2,9 +2,16 @@
 
 RSpec.describe Idv::CaptureDocStatusController do
   let(:user) { build(:user) }
-
   let(:doc_auth_response) do
-    DocAuth::Response.new(
+    Class.new(DocAuth::Response) do
```

Comment:
Why the change from `DocAuth:Response.new` to `Class.new(DocAuth::Response) do...`?

Reply:
@eileen-nava , was testing with mocked response, after rebase not needed anymore.